src/main/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileController.java

```diff
@@ -115,6 +115,7 @@ public ResponseEntity<Void> updateCompanyProfile(
             @Valid @RequestBody CompanyProfile requestBody) {
         DataMapHolder.get()
                 .companyNumber(companyNumber);
+        DataMapHolder.get().contextId(contextId);
```

Comment:
I think this needs to be `DataMapHolder.get().requestId` going off the DataMapHolder class

Reply:
done